modules/pixfutureBidAdapter.js

```diff
@@ -0,0 +1,323 @@
+          rd_stk: bidderRequest.refererInfo.stack.map((url) => encodeURIComponent(url)).join(',')
+        };
+        payload.referrer_detection = refererinfo;
+      }
+
+      if (validBidRequests[0].userId) {
+        let eids = [];
+
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.flocId.id`), 'chrome.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.criteoId`), 'criteo.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.unifiedId`), 'thetradedesk.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.id5Id`), 'id5.io', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.sharedId`), 'thetradedesk.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.identityLink`), 'liveramp.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.liveIntentId`), 'liveintent.com', null);
+        addUserId(eids, utils.deepAccess(validBidRequests[0], `userId.fabrickId`), 'home.neustar', null);
+
+        if (eids.length) {
+          payload.eids = eids;
+        }
+      }
+
+      if (tags[0].publisher_id) {
+        payload.publisher_id = tags[0].publisher_id;
+      }
+
+      const ret = {
+        url: `${hostname}/auc/auc.php`,
+        method: 'POST',
+        options: {withCredentials: false},
+        data: {
+          v: $$PREBID_GLOBAL$$.version,
+          pageUrl: referer,
+          bidId: bidRequest.bidId,
+          auctionId: bidRequest.auctionId,
+          transactionId: bidRequest.transactionId,
+          adUnitCode: bidRequest.adUnitCode,
+          bidRequestCount: bidRequest.bidRequestCount,
+          sizes: bidRequest.sizes,
+          params: bidRequest.params,
+          pubext: payload
+        }
+      };
+      if (bidderRequest && bidderRequest.gdprConsent) {
+        ret.data.gdprConsent = {
+          consentString: bidderRequest.gdprConsent.consentString,
+          consentRequired: (typeof bidderRequest.gdprConsent.gdprApplies === 'boolean') && bidderRequest.gdprConsent.gdprApplies
+        };
+      }
+      return ret;
+    });
+  },
+
+  interpretResponse: function (serverResponse, { bidderRequest }) {
+    serverResponse = serverResponse.body;
+    const bids = [];
+    if (serverResponse.creatives.bids && serverResponse.placements) {
+      serverResponse.placements.forEach(serverBid => {
+        serverBid.creatives.forEach(creative => {
+          const bid = newBid(serverBid, creative, serverBid.placement_id, serverBid.uuid);
+          bid.mediaType = BANNER;
+          bids.push(bid);
+        });
+      });
+    }
+
+    return bids;
+  },
+};
+
+function newBid(serverBid, rtbBid, placementId, uuid) {
+  const bid = {
+    requestId: uuid,
+    cpm: rtbBid.cpm,
+    creativeId: rtbBid.creative_id,
+    currency: 'USD',
+    netRevenue: true,
+    ttl: 300,
+    adUnitCode: placementId
+  };
+  
```

Comment:
When I grab the raw file of this and look, it appears that this line is two spaces rather than just a line break.

Reply:
I  remove all double spaces and created new pull request.
Thank you for help!